ui/src/chat/ChatUnreadAlerts.tsx

```diff
@@ -15,6 +18,27 @@ export default function ChatUnreadAlerts({ whom }: ChatUnreadAlertsProps) {
     useChatState.getState().markRead(whom);
   }, [whom]);
 
+  const navigate = useNavigate();
+  const location = useLocation();
+  // TODO: how to handle replies?
+  const chatKeys = useChatKeys({ replying: false });
+  const goToFirstUnread = useCallback(
+    (b: ChatBrief) => {
+      const { 'read-id': lastRead } = b;
+      if (!lastRead) {
+        return;
+      }
+      // lastRead is formatted like: ~zod/123.456.789...
+      const lastReadBN = bigInt(lastRead.split('/')[1].replaceAll('.', ''));
+      const firstUnread = chatKeys.find((key) => key.gt(lastReadBN));
```

Comment:
The reason you can't do eq here is that the IDs don't actually correlate to the time keys, but this is fine for now we have some other issues to fix before you can associate them correctly I think.